src/fswAlgorithms/orbitControl/lambertPlanner/lambertPlanner.rst

```diff
@@ -0,0 +1,50 @@
+Executive Summary
+-----------------
+This module creates the :ref:`lambertProblemMsgPayload` message that is used as an input to the :ref:`lambertSolver` module. The message includes the required Lambert problem setup information, such as the two position vectors :math:`\mathbf{r}_{1}` and :math:`\mathbf{r}_{2}` as well as a requested time-of-flight. Given a targeted position vector :math:`{}^N\mathbf{r}_{T/N}` with respect to the celestial body at final time :math:`t_{final}` expressed in the inertial frame :math:`N`, the final time :math:`t_{final}`, and the maneuver time :math:`t_{maneuver}`, the module uses a 4th order Runge-Kutta (RK4) to propagate the current state (obtained from the :ref:`NavTransMsgPayload` navigation input message) to obtain the estimated position at maneuver time. This estimated position corresponds to the first position vector :math:`\mathbf{r}_{1}` of Lambert's problem, while :math:`{}^N\mathbf{r}_{T/N}` corresponds to the second position vector :math:`\mathbf{r}_{2}`.
```

Comment:
I think we want to keep these lines shorter

Reply:
Done.